pages/common/git-add.md

```diff
@@ -11,6 +11,10 @@
 
 `git add -A`
 
+- Add all files (It's Add all file in one commend):
```

Comment:
Can you update the description to explain what is the difference between this and `git add -A`?
https://www.logilax.com/git-add-dot-vs-git-add-a/